cluster/images/etcd/rollback/rollback.go

```diff
@@ -0,0 +1,335 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"os"
+	"path"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/coreos/etcd/etcdserver"
+	pb "github.com/coreos/etcd/etcdserver/etcdserverpb"
+	"github.com/coreos/etcd/etcdserver/membership"
+	"github.com/coreos/etcd/mvcc/backend"
+	"github.com/coreos/etcd/mvcc/mvccpb"
+	"github.com/coreos/etcd/pkg/pbutil"
+	"github.com/coreos/etcd/pkg/types"
+	"github.com/coreos/etcd/raft/raftpb"
+	"github.com/coreos/etcd/snap"
+	"github.com/coreos/etcd/store"
+	"github.com/coreos/etcd/wal"
+	"github.com/coreos/etcd/wal/walpb"
+	"github.com/golang/glog"
+)
+
+var (
+	migrateDatadir string
+	ttl            time.Duration
+)
+
+func init() {
+	flag.StringVar(&migrateDatadir, "data-dir", "", "Path to the data directory")
+	flag.DurationVar(&ttl, "ttl", time.Hour, "TTL of event keys")
+	flag.Parse()
+}
```

Comment:
Can you please change the lines above into:

var (
  migrateDataDir = flag.String("data-dir", "", "Path to the data directory")
  ttl = flag.Duration("ttl", time.Hour, "TTL of event keys (default 1 hour)")
)
[This is the convention we are using for small tools.]